data_for_test.go

```diff
@@ -375,7 +375,7 @@ var suites = []FixtureSuite{
 					WithTraceparentIgnoreEnv(true).
 					WithTraceparentPrint(true).
 					WithTraceparentPrintExport(true).
-					WithTraceparentRequired(true).
+					WithTraceparentRequired(false).
```

Comment:
It was a bug that this didn't break before. The example TP file wasn't ever available, so --tp-required wasn't working right in this case. It's fixed in this PR so the test broke.